packages/website-backend/src/services/Configuration.ts

```diff
@@ -16,5 +17,11 @@ export default class Configuration {
       util.optionalEnvVar('NODE_ENV', 'production') === 'development'),
       (this.baseUrl = util.requiredEnvVar('STRYKER_DASHBOARD_BASE_URL'));
     this.jwtSecret = util.requiredEnvVar('JWT_SECRET');
+
+    if (this.isDevelopment) {
```

Comment:
I think `isDevelopment` is unnecessary; always use `STRYKER_DASHBOARD_BASE_URL` to set cors. This should also work locally since there this env variable is also set..